test/external-modules/apm-integration/src/main/java/org/elasticsearch/test/apmintegration/TestMeterUsages.java

```diff
@@ -34,6 +34,9 @@ public TestMeterUsages(MeterRegistry meterRegistry) {
         this.longHistogram = meterRegistry.registerLongHistogram("testLongHistogram", "test", "unit");
         meterRegistry.registerDoubleGauge("testDoubleGauge", "test", "unit", doubleWithAttributes::get);
         meterRegistry.registerLongGauge("testLongGauge", "test", "unit", longWithAttributes::get);
+
+        meterRegistry.registerLongAsyncCounter("testAsyncLongCounter", "test", "unit", longWithAttributes::get);
+        meterRegistry.registerDoubleGauge("testAsyncDoubleCounter", "test", "unit", doubleWithAttributes::get);
```

Comment:
Shouldn't this be `registerDoubleAsyncCounter`?

Reply:
good catch. yes